release-note.html

```diff
@@ -0,0 +1,113 @@
+<!--
+  ~  Copyright (c), WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~  WSO2 Inc. licenses this file to you under the Apache License,
+  ~  Version 2.0 (the "License"); you may not use this file except
+  ~  in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing,
+  ~  software distributed under the License is distributed on an
+  ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~  KIND, either express or implied.  See the License for the
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  -->
+
+<!DOCTYPE html
+        PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+    <meta content="text/html; charset=iso-8859-1" http-equiv="content-type"/>
+    <title>Choreo Connect v0.9.0-Alpha</title>
+</head>
+<body>
+<div id="main-content">
+    <div>
+        <h1>Choreo Connect v0.9.0-Alpha is Released</h1>
+        <h2>What's new in Choreo Connect 0.9.0-Alpha</h2>
+        <ul>
+            <li>
+                <p>This is the alpha release of the Choreo Connect 0.9.0</p>
+            </li>
+            <li>
+                <p>Microservices have become the norm for modern application architecture. Workloads of modern
+                    applications are<br>
+                    spread across many groups of microservices, cloud services and legacy services. The characteristics
+                    and<br>
+                    behaviors of such heterogeneous services have a massive diversity. Such as authentication
+                    mechanisms, message<br>
+                    formats, high availability factors and so on. The Choreo-Connect is designed to expose heterogeneous<br>
+                    microservices as APIs to end consumers using a common API interface based on the Open API
+                    Specification. This<br>
+                    helps expose microservices using a unified interface to external consumers, internal consumers and
+                    partners. It<br>
+                    applies the common quality of service attributes on API requests such as security, rate limiting and
+                    analytics<br>
+                    and also offers a wide range of features which helps organizations to deploy APIs microservice
+                    architectures<br>
+                    efficiently.</p>
+            </li>
+        </ul>
+        <h2>Features</h2>
+        <ul>
+            <li>Choreo Connect provides <a
+                    href="https://github.com/wso2/product-microgateway/issues?q=is%3Aissue+project%3Awso2%2Fproduct-microgateway%2F8+is%3Aclosed+label%3A%22Type%2FNew+Feature%22">features</a>
+                for API Management.
```

Comment:
Is this part required

Reply:
We can just named it as features and point to the link if that's okay

Reply:
Modified. Please have a look.